services/hapi/src/index.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+const updateBpStats = require('./libs/sync-bp-stats')
+const updateUserRatings = require('./libs/sync-user-rating')
+
+const Hapi = require('@hapi/hapi')
+
+const init = async () => {
+  const server = Hapi.server({
+    port: 3005,
+    host: '0.0.0.0'
+  })
+
+  server.route({
+    method: 'GET',
+    path: '/',
+    handler: function() {
+      return 'Hello World!'
+    }
+  })
+
+  server.route({
+    method: 'GET',
+    path: '/ratebp',
+    handler: request => {
+      const bp = request.query.producer
+      if (bp) {
+        updateBpStats(bp)
```

Comment:
@tetogomez  @rubenabix  Here we update the stats table with the new values for the BP rating so that it reflects on the frontend.